### PR TITLE
Dev -> Master for 2.1 release

### PR DESCRIPTION
**Pull request by drpatelh**

### CODE REVIEW

**Review by JoseEspinosa — APPROVED** [review]
Looks awesome as always! 🎸 
Just made a small suggestion!

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,42 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [[2.1](https://github.com/nf-core/rnaseq/releases/tag/2.1)] - 2021-06-11
+
+### Enhancements & fixes
+
+* Removed workflow to download data from public databases in favour of using [nf-core/fetchngs](https://nf-co.re/fetchngs)
+* Added Pangolin VOC scorpio calls to default variant calling summary metrics
+* Dashes in sample names will be converted to underscores to avoid issues when creating the summary metrics via QUAST
```
[the remark this answers is not in the document]
Comment (drpatelh) [reply CHANGELOG.md:12]: Fixed https://github.com/nf-core/viralrecon/pull/202/commits/aaec38e34095741bafe18d71410fc747b56ab8e5 🦅 👀

**Review by heuermh — APPROVED** [review]
LGTM!

**Comment by JoseEspinosa** [comment]
Sorry, I closed it by mistake on the phone app... 🤦 reopened 1 sec after but...

**Pull request merged**